src/landing-page/components/LandingPage/index.tsx

```diff
@@ -0,0 +1,301 @@
+  z-index: -1;
+
+  & > video {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    // Fixes a bug where a single pixel black outline would appear around the video.
+    clip-path: inset(1px 1px);
+  }
+`;
+
+const ContentContainer = styled(FlexColumn)`
+  max-width: 1060px;
+  width: calc(90vw - 40px);
+  margin: auto;
+  padding: 0 20px;
+`;
+
+const FeatureHighlightsContainer = styled.li`
+  display: grid;
+  width: 100%;
+  grid-template-rows: repeat(2, auto);
+  grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
+  padding: 0;
+  justify-content: space-evenly;
+  gap: 10px;
+  margin: 20px 0;
+`;
+
+const FeatureHighlightsItem = styled(FlexColumn)`
+  display: grid;
+  grid-template-rows: subgrid;
+  grid-row: span 2;
+
+  & > h3 {
+    font-weight: 600;
+  }
+`;
+
+const Divider = styled.hr`
+  display: block;
+  width: 100%;
+  height: 1px;
+  background-color: var(--color-borders);
+  border-style: none;
+`;
+
+const ProjectList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 0;
+  margin-top: 0;
+
+  // Add a pseudo-element line between cards
+  & > li:not(:first-child)::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 1px;
+    background-color: var(--color-borders);
+    margin-bottom: 10px;
+  }
+`;
+
+const ProjectCard = styled.li`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  gap: 12px;
+
+  & h3 {
+    font-weight: 600;
+  }
+`;
+
+const DatasetList = styled.ol`
+  padding: 0;
+  width: 100%;
+  display: grid;
+  // Use grid + subgrid to align the title, description, and button for each horizontal
+  // row of cards. repeat is used to tile the layout if the cards wrap to a new line.
+  grid-template-rows: repeat(3, auto);
+  grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
+  justify-content: space-around;
+  gap: 10px 20px;
+`;
+
+const DatasetCard = styled.li`
+  display: grid;
+  grid-template-rows: subgrid;
+  grid-row: span 3;
+  min-width: 180px;
+  padding: 5px;
+
+  & > h4 {
+    text-align: center;
+    display: grid;
+    margin: 0;
+  }
+  & > p {
+    text-align: center;
+    display: grid;
+  }
+  & > a {
+    margin: auto;
+    display: grid;
+  }
+`;
+
+const InReviewFlag = styled(FlexRowAlignCenter)`
+  border-radius: 4px;
+  padding: 1px 6px;
+  background-color: var(--color-flag-background);
+  height: 22px;
+  flex-wrap: wrap;
+
+  & > p {
+    color: var(--color-flag-text);
+    font-size: 10px;
+    font-weight: 700;
+    white-space: nowrap;
+  }
+`;
+
+type LandingPageProps = {
+  load: (args: ViewerArgs) => void;
+};
+
+export default function LandingPage(props: LandingPageProps): ReactElement {
+  // Rendering
+
+  // TODO: Should the load buttons be link elements or buttons?
+  // Currently both the link and the button inside can be tab-selected.
+  const renderDataset = (dataset: DatasetEntry, index: number): ReactElement => {
+    return (
+      <DatasetCard>
```

Comment:
`index` is unused here, but this function is used in a `map` and the component it renders may want a `key` property. Looks like `index` is used for that in the equivalent file in nucmorph-colorizer - should it be used here too?

Reply:
That makes sense, thank you!